en/matrix.md

```diff
@@ -0,0 +1,417 @@
+
+          <td class="col-multipartytxn" style="display: none;">
+          {% include functions/matrix-cell-1-input.md state=tool.feature.coinjoin %}
+
+          <td class="col-multipartytxn" style="display: none;">
+          {% include functions/matrix-cell-4-input.md input="BIP78 - " input2=tool.feature.payjoin_BIP78 input3=tool.feature.payjoin_alternate input4=tool.feature.payjoin_alternate_name %}
+
+          <td class="col-pc_silentpayments" style="display: none;">
+          {% include functions/matrix-cell-1-input.md state=tool.feature.payment_code %}
+
+          <td class="col-pc_silentpayments" style="display: none;">
+          {% include functions/matrix-cell-1-input.md state=tool.feature.silent_payments_BIP352 %}
+
+          <td class="col-pc_silentpayments" style="display: none;">
+          {% include functions/matrix-cell-1-input.md state=tool.feature.silent_payments_dns %}
+
+          <td class="col-lightning" style="display: none;">
+          {% include functions/matrix-cell-1-input.md state=tool.feature.lightning_11 %}
+
+          <td class="col-lightning" style="display: none;">
+          {% include functions/matrix-cell-1-input.md state=tool.feature.lightning_12 %}
+
+          <td class="col-lightning" style="display: none;">
+          {% include functions/matrix-cell-1-input.md state=tool.feature.lightning_12_dns %}
+
+          <td class="col-lightning" style="display: none;">
+          {% include functions/matrix-cell-1-input.md state=tool.feature.lightning_lnurl %}
+
+          <td class="col-lightning" style="display: none;">
+          {% include functions/matrix-cell-1-input.md state=tool.feature.lightning_lnurl_hrla %}
+
+        </tr>
+      {% endif %}
+
+  {% endfor %}
+
+  </tbody>
+  </table>
+
+  <table>
+
+  <tr style="background-color: transparent;">
+  <td colspan="15" style="text-align:left;font-size:14px;border-style:hidden;" >
+  <span class="bold">Notes:</span><br/>
+  &#x1F538; Hover over a column header to see that column's details.<br/>
+  &#x1F538; Features with alternate implementations are denoted with <font color="orange" size="3">*</font>, hover for details.<br/>
+  </td>
+
+  </tr>
+  <tr style="background-color: transparent;">
+  <td colspan="15" style="text-align:left;font-size:14px;border-style:hidden;" >
+  <span class="bold">Approach:</span><br/>
+  &#x1F538; Bitcoinops has selected the initial features within the matrix, focusing on interoperability.  <br/>
+  &#x1F538; The matrix will be split by category once enough results are collected.<br/>
+  &#x1F538; New feature as well as Product/Service requests are welcome!  Open a PR in the <a href="https://github.com/bitcoinops/bitcoinops.github.io">Optech Github</a>.<br/>
+  </td>
+  </tr>
+
+  </table>
+
+<br/>
+
+<br/>
+_Contributions and corrections are welcome. Please see the [contibuting
+guidelines](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/CONTRIBUTING.md)
```

Comment:
Nit: It would be better if this linked to the Compatibility Matrix Data section of the contributing guide.

```suggestion
guidelines](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/CONTRIBUTING.md#compatibility-matrix-data)
```